ArcGISRuntimeSDKQt_CppSamples/Analysis/TraceUtilityNetwork/TraceUtilityNetwork.cpp

```diff
@@ -63,7 +66,15 @@ FindFeaturesUtilityNetwork::FindFeaturesUtilityNetwork(QObject* parent /* = null
 
   m_lineFeatureTable = new ServiceFeatureTable(QUrl("https://sampleserver7.arcgisonline.com/arcgis/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer/115"), this);
   m_lineLayer = new FeatureLayer(m_lineFeatureTable, this);
-  m_lineLayer->setRenderer(new SimpleRenderer(m_lineSymbol, this));
+
+  // create unique renderer
+  m_uniqueValueRenderer = new UniqueValueRenderer(this);
+  m_uniqueValueRenderer->setFieldNames(QStringList("ASSETGROUP"));
+  createUniqueValue(QString("Medium Voltage"), m_mediumVoltageSymbol, 5);
```

Comment:
this function name is a little misleading since it is creating and adding. maybe it should return a unique value and you append here? Or change name to be more accurate?

Reply:
I like the idea of returning and appending. I'll do that!